openid-4-verifiable-presentations-1_0.md

```diff
@@ -1864,6 +1911,30 @@ Wallets SHOULD NOT fetch URLs provided by the Verifier in a request that are unk
 
 Ecosystems that plan to leverage the trusted authorities mechanisms SHOULD make sure that the privacy properties of the mechanisms they choose to support matches with the desired privacy properties of the overall ecosystem.
 
+## Use, Retention and Disclosure Limitation {#use_retention_and_disclosure_limitation}
+
+### Unobservability by Wallets {#unobservability_by_wallets}
+
+    
+*   **User-Only Visibility**: Transaction history and data within the wallet should not be accessible to anyone other than the user, unless the user has given consent or there is another legal basis to do so.
+    
+*   **Prevention of Profiling and Tracking:** The wallet's design incorporates data minimization principles, ensuring that only essential information is shared during interactions. This limits opportunities for profiling or linking user activities across different services
+
+## Accuracy and Quality {#accuracy_and_quality}
+
+* Verifiers should establish a means to detect status change (e.g. revocation) of the obtained credentials through the OpenID4VP protocol. (e.g., through StatusList.) 
+    
+## Openness, Transparency and Notice {#openness_transparency_and_notice}
+
+*   Wallet should make its privacy notices readily available to the user. 
+        
+## Information Security {#information_security}
+
+*   Wallet providers and verifiers should choose the options in the protocols described in this document so that the data in transit is integrity, confidentiality and availability protected. 
```

Comment:
```suggestion
*   Wallet providers and Verifiers should choose the options in the protocols described in this document so that the data in transit is integrity, confidentiality and availability protected. 
```